structured/structured.proto

```diff
@@ -59,14 +59,15 @@ message IndexDescriptor {
   optional string name = 1 [(gogoproto.nullable) = false];
   optional uint32 id = 2 [(gogoproto.nullable) = false, (gogoproto.customname) = "ID"];
   optional bool unique = 3 [(gogoproto.nullable) = false];
+  optional bool primary = 4 [(gogoproto.nullable) = false];
```

Comment:
Makes sense. Ignore the latest push. Let me restructure the code to do just that.